### PR TITLE
Fixed inconsistent first character capitalization.

### PR DESCRIPTION
**Pull request by dhruvmaindola227**
In some bullet points there were words where first character wasn't in uppercase and in some they were. Fixed the inconsistency.

### CODE REVIEW

**Comment by dhruvmaindola227** [comment]
@parispittman  @alisondy can you please review the grammatical changes? I'm a newbie in open source.

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

**Review by gabrielavelino — APPROVED** [review]
The titles looks more consistent starting with uppercase letters. The changes that you made, i think they not affect the grammatical part.

**Comment by dhruvmaindola227** [comment]
> The titles looks more consistent starting with uppercase letters. The changes that you made, i think they not affect the grammatical part.

I understand that it doesn't affect grammatically but it looked inconsistent in the documentation so i tried to change it. I'm very new to open source and figured such small changes would be harmless.

File: contributors/guide/contributing.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Contributing to Kubernetes"
-weight: 4
-description: |
+Title: "Contributing to Kubernetes"
+Weight: 4
+Description: |
```
Comment (mrbobbytables) [inline contributors/guide/contributing.md:4]: Thanks! just one thing
This is metadata headers and should remain lowercase. If you take a peek at all the files in the guide, they'll all be lowercase 👍

**Comment by gabrielavelino** [comment]
> > The titles looks more consistent starting with uppercase letters. The changes that you made, i think they not affect the grammatical part.
> 
> I understand that it doesn't affect grammatically but it looked inconsistent in the documentation so i tried to change it. I'm very new to open source and figured such small changes would be harmless.



> > The titles looks more consistent starting with uppercase letters. The changes that you made, i think they not affect the grammatical part.
> 
> I understand that it doesn't affect grammatically but it looked inconsistent in the documentation so i tried to change it. I'm very new to open source and figured such small changes would be harmless.

i am new too, i agreed with the changes. it's nice to have new people around here too =).

File: contributors/guide/contributing.md (hunk 1 above)
Comment (dhruvmaindola227) [reply contributors/guide/contributing.md]: > Thanks! just one thing This is metadata headers and should remain lowercase. If you take a peek at all the files in the guide, they'll all be lowercase 👍

Thanks for sharing that! I have changed it back to how it was.

**Comment by dhruvmaindola227** [comment]
@mrbobbytables I have removed the metadata headers change and remaining changes are still there.

**Review by mrbobbytables — COMMENTED** [review]
Sorry, this fell through the cracks^^;;;
/approve

**Pull request merged**